src/modules/LanItems.rb

```diff
@@ -430,7 +430,13 @@ def ReplaceItemUdev(replace_key, new_key, new_val)
       operator = new_key == "NAME" ? "=" : "=="
       current_rule = getUdevFallback
```

Comment:
What I don't link about this method is that calls getUdevFallback which creates a rule if not exist inside a method called ReplaceItemUdev, and then calling again RemovekeFromUdevRule over (getUdevFallback) and then comparing again new rule with current rule is alitle strange for me.